zarr/meta.py

```diff
@@ -10,11 +10,14 @@
 from zarr.errors import MetadataError
 
 
-def decode_metadata(b):
+ZARR_FORMAT = 2
+
+
+def decode_array_metadata(b):
     s = text_type(b, 'ascii')
     meta = json.loads(s)
     zarr_format = meta.get('zarr_format', None)
-    if zarr_format != 1:
+    if zarr_format != ZARR_FORMAT:
```

Comment:
I guess there are probably few users of v1? Might be worth considering making zarr backwards compatible (and/or releasing an upgrade script), at least for future version changes.